src/In.ProjectEKA.OtpService/Otp/OtpController.cs

```diff
@@ -37,7 +38,7 @@ private ActionResult ReturnServerResponse(OtpResponse otpResponse)
                 ResponseType.OtpInvalid => BadRequest(otpResponse),
                 ResponseType.OtpExpired => Unauthorized(otpResponse),
                 ResponseType.OtpGenerationFailed => BadRequest(otpResponse),
```

Comment:
If the generation failed, isn't it a server problem?

Reply:
Should the OTP API be referenced as /otp/link or can't that just be /otp?
Should the OTP VERIFY API be referenced as /otp/verify or can't that just be /otp/{sessionID}/verify?

Reply:
Should the OTP API be referenced as /otp/link or can't that just be /otp: Changed